startbootstrap-sb-admin-gh-pages/tables.html

```diff
@@ -673,6 +210,65 @@ <h5 class="modal-title" id="exampleModalLabel">Ready to Leave?</h5>
     </div>
   </div>
 
+  <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js'></script>
+  <script type="text/javascript">
+    function refresh(){
+      let xhr = new XMLHttpRequest();
+      var url = "http://localhost:5002/all";
+      xhr.open('GET', url)
+      xhr.send()
+      xhr.onreadystatechange = function() {
+        if (xhr.readyState==4 && xhr.status==200) {
+          var data = JSON.parse(xhr.responseText);
+          console.log(data.length)
+
+          // tableRef = document.getElementById('table')
+          // var newRow   = tableRef.insertRow();
+          // // Insert a cell in the row at index 0
+          // var cell0 = newRow.insertCell();
+          // var cell1 = newRow.insertCell();
+          // var cell2  = newRow.insertCell();
+          // var cell3  = newRow.insertCell();
+          // var cell4  = newRow.insertCell();
+          // var cell5  = newRow.insertCell();
+          // var cell6  = newRow.insertCell();
+          // var cell7  = newRow.insertCell();
+          for (i = 0; i< data.length; i++){
+            tableRef = document.getElementById('table')
+            var newRow   = tableRef.insertRow();
+          // Insert a cell in the row at index 0
+            var cell0 = newRow.insertCell();
```

Comment:
can you use more meaningful variables names instead of 'cellX'... For example, cashOutCell, buyInCell, etc.